cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -80,6 +80,7 @@ MSG(object_does_not_provide_read_access_to_columns,
 MSG(object_does_not_provide_write_access_to_columns,
     "Given object does not provide write access to columns")
 MSG(unsupported_conversion_types, "Unsupported conversion types")
+MSG(invalid_indices, "Invalid indices")
```

Comment:
Very unclear error message. Remember that these are messages that your users see. Please be more specific

Reply:
The different error massage were created, it was fixed